scss/_navbar.scss

```diff
@@ -139,13 +139,49 @@
 // Keep as a separate element so folks can easily override it with another icon
 // or image file as needed.
 .navbar-toggler-icon {
+  position: relative;
   display: inline-block;
-  width: 1.5em;
-  height: 1.5em;
-  vertical-align: middle;
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 100%;
+  width: $navbar-toggler-font-size;
+  height: $border-width * 2;
+  vertical-align: super;
```

Comment:
I'd keep `middle` as a value, and simply add `top: -#{$border-width}` to center icon.